module-2/backend/Liam_barstad/week_one.txt

```diff
@@ -0,0 +1,92 @@
+List the five common HTTP verbs and what the purpose is of each verb.
+  - get: to fetch views content
+  - post: to upload content and/or add to database
+  - delete: to delete content and/or remove from database
+  - put: ??
+  - update: to rewrite content and/or replace elements in the database
+
+What is Sinatra?
+  Sinatra is a hosting service for ruby-based websites
+
+What is MVC?
+  The model-view-controller style of creating ruby-based websites, where the controller files navigate
+  through webpages, stored in views. It can also add erb content, initiated through models. The models
+  stores and interprets data stored in the database.
```

Comment:
Careful with the language here:

* I wouldn't say that the controller `navigates through webpages`. Our views aren't really `webpages` per se, they're templates that we use to create pages.
* Similarly, we don't `add erb content`. ERB allows us to execute Ruby code in our HTML templates.
* While an instance of the model may store data temporarily, I would probably say that it accesses and interprets data stored in the database.